packages/ubus-lime-batman-adv/Makefile

```diff
@@ -13,7 +13,7 @@ define Package/$(PKG_NAME)
   MAINTAINER:=Marcos Gutierrez <gmarcos87@gmail.com>
   SUBMENU:=3. Applications
   TITLE:=B.A.T.M.A.N.-Adv ubus status module
-  DEPENDS:= +lua +libubox-lua +libubus-lua +kmod-batman-adv +batctl +luci-lib-nixio +lime-system
+  DEPENDS:= +lua +libubox-lua +libubus-lua +luci-lib-nixio +lime-system
```

Comment:
IT is ok to remove the dependency here?

Reply:
just stumbled over this commit. it's definitive not ok to remove this dependency, afais kmod-batman-adv and batctl are not pulled anywhere else and to me it seems like a module called ubus-lime-batman-adv really needs both dependencies to work ;)

Reply:
@gmarcos87 would you like to argument on this?

Reply:
What I want to do is that this package will be selected automatically if lime-proto-batadv is selected. How would that be?